fl4health/parameter_exchange/parameter_selection_criteria.py

```diff
@@ -0,0 +1,117 @@
+import math
+from typing import Dict, List, Tuple
+
+import torch
+import torch.nn as nn
+from flwr.common.typing import NDArrays
+from torch import Tensor
+
+
+# Selection criteria functions for selecting entire layers. Should be used with the DynamicLayerExchanger class.
+def _calculate_drift_norm(t1: torch.Tensor, t2: torch.Tensor, normalize: bool) -> float:
+    t_diff = (t1 - t2).float()
+    drift_norm = torch.linalg.norm(t_diff)
+    if normalize:
+        drift_norm /= torch.numel(t_diff)
+    return drift_norm.item()
+
+
+def select_layers_by_threshold(
+    model: nn.Module, initial_model: nn.Module, threshold: float, normalize: bool, select_drift_more: bool
+) -> Tuple[NDArrays, List[str]]:
+    """
+    Return those layers of model that deviate (in l2 norm) away from corresponding layers of
+    self.initial_model by at least (or at most) self.threshold.
+    """
+    layer_names = []
+    layers_to_transfer = []
+    initial_model_states = initial_model.state_dict()
+    model_states = model.state_dict()
+    for layer_name, layer_param in model_states.items():
+        layer_param_past = initial_model_states[layer_name]
```

Comment:
I move that we call this variable `ghost_of_layer_params_past`. @jewelltaylor, do you second this motion?